src/background/background.coffee

```diff
@@ -74,7 +74,7 @@ handleRequestFromContentScript = (request, sender, sendResponse) ->
     if script.indexOf('Shim') == -1
       trackController(request.host)
     chrome.tabs.executeScript(sender.tab.id, {file: script})
-    if request.host.indexOf("youtube") > -1
+    if request.host.indexOf("youtube") > -1 || request.host.indexOf("connect.monstercat") > -1
```

Comment:
Nope, I'm just a fan.

This could be addressed by looking at whether the controller variable is defined/a controller and loading the shim if it's not